src/analysis/normed_space/inner_product.lean

```diff
@@ -1833,6 +1838,55 @@ begin
   rwa [h, sup_comm, bot_sup_eq] at this,
 end
 
+/-- A point in `K` with the orthogonality property (here characterized in terms of `Kᗮ`) must be the
+orthogonal projection. -/
+lemma eq_orthogonal_projection_of_mem_orthogonal {K : submodule 𝕜 E} [complete_space K]
```

Comment:
Should we have a variables line here? I see many lemmas starting with `{K : submodule 𝕜 E} [complete_space K]`

Reply:
I added a variables line and made the appropriate changes all through the last 400 lines of the file (orthogonal projections and orthogonal complements). It was quite a big job so I won't take advantage of your `bors d+`; please check it!